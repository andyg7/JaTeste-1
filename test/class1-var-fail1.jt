func int main()
{

	struct house *my_house;
	int price;
	int vol;

	my_house->set_price(100);
	my_house->set_height(88);
	my_house->set_width(60);
	my_house->set_length(348);


	return 0;
}

struct house {
	int price;
	int height;
	int width;
	int length;

	method void set_price(int x)
	{
		pricee = x;
	}

	method void set_height(int x)
	{
		height = x;
	}

	method void set_width(int x)
	{
		width = x;
	}

	method void set_length(int x)
	{
		length = x;
	}

	method int get_price()
	{
		return price;
	}

	method int get_volumne()
	{
		int temp;
		temp = height * width * length;
		return temp;
	}


};
