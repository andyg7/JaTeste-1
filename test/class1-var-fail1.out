Scanned
Parsed
Fatal error: exception Exceptions.UndeclaredVariable("pricee")
